src/main/java/homeworks/andrea/secondAssignment/ParentClassOne.java

```diff
@@ -0,0 +1,39 @@
+package homeworks.andrea.secondAssignment;
+
+public abstract class ParentClassOne implements InterfaceABC {
+
+    private int a;
+    private String b;
+
+    public int getA() {
+        return a;
+    }
+
+    public void setA(int a) {
+        this.a = a;
+    }
+
+    public String getB() {
+        return b;
+    }
+
+    public void setB(String b) {
+        this.b = b;
+    }
+
+    @Override
```

Comment:
Therefore there's no need for an override here nor any other place in this class.